src/NzbDrone.Core/Housekeeping/Housekeepers/CleanupOrphanedPendingReleases.cs

```diff
@@ -18,9 +18,9 @@ public void Clean()
             mapper.ExecuteNonQuery(@"DELETE FROM PendingReleases
                                      WHERE Id IN (
                                      SELECT PendingReleases.Id FROM PendingReleases
-                                     LEFT OUTER JOIN Series
-                                     ON PendingReleases.SeriesId = Series.Id
-                                     WHERE Series.Id IS NULL)");
+                                     LEFT OUTER JOIN Movies
+                                     ON PendingReleases.MovieId = MovieId.Id
+                                     WHERE MovieId.Id IS NULL)");
```

Comment:
This needs to be Movies.Id IIRC. Please run the Housekeeping commands from System -> Tasks and see if it completes sucessfully.